Proftpd expects that /etc/shells is populated for these tests, which is not the case in masterdirs
--------
--- a/tests/api/auth.c	2023-11-06 17:04:38.205368713 +0100
+++ -	2023-11-06 17:05:20.578137395 +0100
@@ -2224,7 +2224,6 @@
   tcase_add_test(testcase, auth_get_anon_config_test);
   tcase_add_test(testcase, auth_chroot_test);
   tcase_add_test(testcase, auth_banned_by_ftpusers_test);
-  tcase_add_test(testcase, auth_is_valid_shell_test);
   tcase_add_test(testcase, auth_set_groups_test);
   tcase_add_test(testcase, auth_get_home_test);
   tcase_add_test(testcase, auth_set_max_password_len_test);
